### ISSUE TITLE
Implement cache buster on generated JavaScripts' references

### ISSUE THREAD

**Issue by fabriciomurta** [bridgedotnet/Live#8]
@vladsch complained more than once about having to refresh the page several times in order to have a result reproduced. In fact, this might even mislead code evaluation/debugging when trying several things.

Setting the includes as below might suffice:
```
<script src="bridge.min.js?GUID_HERE"></script>`
```

**Comment by DaniilVeriga** [comment]
It was done in this commit:
https://github.com/bridgedotnet/Live/commit/8519f124f710d0a04ade2a25c3910020a658fd5b

**Comment by fabriciomurta** [comment]
Shouldn't we generate a random stamp? Was thinking on a base64 representation of the time YYYYMMDDHHMMSS =P
The ru-RU.js also might have a cache buster in the event it changes?

Or maybe just update the cache buster every commit to the repo? But update it "automatically", for example with JavaScripts' MD5 sum.

**Comment by DaniilVeriga** [comment]
Yes, sounds a little overkill to me. The cache buster should only be updated when we update Bridge in Live. Though, there is one great plus in your suggestions - it guarantees a cache buster won't be forgotten to update. But, personally, I would leave the things as it is now. There are more important problems with Live Bridge (and others) currently. I would focus on them. As for updating cache buster, I think there is a bigger task that includes updating a cache buster as well - automation of updating to a new Bridge release. Currently, it is done manually. It is probably a task for Bridge v2. Live Bridge will be needed to review. After it stabilizes we can think on automation.
